docs/api-table.js

```diff
@@ -1050,15 +1052,20 @@ export const vue = {
   Field: {
     required: [],
     props: [
-      ['label', 'string', '', ''],
+
+      ['label', 'string', '', 'The content to display as the label'],
       ['hint', 'string', '', 'The string can contain HTML.'],
-      ['invalid', 'boolean', '', ''],
+      ['invalid', 'boolean', '', 'Renders the field in an invalid state. Often paired with `hint`to provide feedback about the error.'],
```

Comment:
```suggestion
      ['invalid', 'boolean', '', 'Renders the field in an invalid state. Often paired with `hint` to provide feedback about the error.'],
```

Reply:
Updated in this commit: [eef7e63](https://github.com/warp-ds/tech-docs/pull/216/commits/eef7e638298d392830bfc81184eb31c19f867b0a)